lib/ddtrace/contrib/qless/qless_job.rb

```diff
@@ -0,0 +1,70 @@
+require 'ddtrace/ext/app_types'
+require 'ddtrace/contrib/analytics'
+require 'qless'
+
+module Datadog
+  module Contrib
+    module Qless
+      # Uses Qless job hooks to create traces
+      module QlessJob
+        def around_perform(job)
+          return super unless datadog_configuration && tracer
+          tracer.trace(Ext::SPAN_JOB, span_options) do |span|
+            span.resource = job.klass_name
+            span.span_type = Datadog::Ext::AppTypes::WORKER
+            span.set_tag(Ext::TAG_JOB_ID, job.jid)
+            span.set_tag(Ext::TAG_JOB_QUEUE, job.queue_name)
+            span.set_tag(Ext::TAG_JOB_TAGS, job.tags)
```

Comment:
I was reading about the purpose of [`job.tags`](https://github.com/seomoz/qless#features) and I noticed they state:
> Tagging / Tracking -- Some jobs are more interesting than others. Track those jobs to get updates on their progress. Tag jobs with meaningful identifiers to find them quickly in the UI.

This _seems_ to me like tags can carry uniquely identifiable user information (like email addresses). In contrast with `job.queue_name` (setup time configuration) or `job.jid` (opaque UID), it think that `job.tags` can contain information that the user might not want to have stored by default.

Would you say I'm understanding the tags concept correctly? If so, we should gate this data collection the same way you did for `job.data` already (thank you for that one, btw!).

Reply:
done